app/components/SceneSelector.vue.ts

```diff
@@ -29,6 +29,10 @@ export default class SceneSelector extends Vue {
 
   searchQuery = '';
 
+  addSceneTooltip = $t('Add a new Scene.');
+  removeSceneTooltip = $t('Remove Scene.');
+  showTransitionsTooltip = $t('Open the Scene Transitions.');
```

Comment:
I appreciate you copying the tone/style of the source properties tooltip here.  That said, I think our current one is poorly worded so I'm wondering if we can phrase this a little better.  I would probably call this something more like "Edit Scene Transitions."